### PR TITLE
PAR-645: Added initial form for invite. To be added to the flow later.

### PR DESCRIPTION
**Pull request by stephenb01**

### CODE REVIEW

File: web/modules/features/par_partnership_flows/src/Form/ParPartnershipFlowsInviteForm.php [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+use Drupal\par_data\Entity\ParDataPerson;
+use Drupal\par_flows\Form\ParBaseForm;
+use Drupal\user\Entity\User;
+
+/**
+ * Class InviteByEmailBlockForm.
+ *
+ * @package Drupal\invite\Form
+ */
+class ParFlowTransitionInviteForm extends ParBaseForm {
```
Comment (kalpaitch) [inline web/modules/features/par_partnership_flows/src/Form/ParPartnershipFlowsInviteForm.php]: Nothing says Ctrl+C, Ctrl+V more :)

File: web/modules/features/par_partnership_flows/src/Form/ParPartnershipFlowsInviteForm.php [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId() {
+    return 'par_partnership_invite_organisation';
```
Comment (kalpaitch) [inline web/modules/features/par_partnership_flows/src/Form/ParPartnershipFlowsInviteForm.php]: It probably won't be just for inviting organisation members, let's just make it `par_partnership_invite`

File: web/modules/features/par_partnership_flows/src/Form/ParPartnershipFlowsInviteForm.php [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+        if ($par_data_partnership->personIsAuthorityMember($authority_person)) {
+          $this->loadDataValue("authority_member", $par_data_person->retrieveStringValue('email'));
+          $authority_person_name = $authority_person->getFullName();
+          break 1;
+        }
+      }
+      $message_body = <<<HEREDOC
+Dear {$par_data_person->getFullName()},
+
+I'm writing to ask you to check and update if necessary the information held about your business in the Primary Authority Register. To do this, please follow this link:
```
Comment (kalpaitch) [inline web/modules/features/par_partnership_flows/src/Form/ParPartnershipFlowsInviteForm.php]: Will need to get a new block of copy as the default message. Best ask Sophie.

File: web/modules/features/par_partnership_flows/src/Form/ParPartnershipFlowsInviteForm.php (hunk 1 above)
Comment (stephenb01) [reply web/modules/features/par_partnership_flows/src/Form/ParPartnershipFlowsInviteForm.php]: Yeah spotted that just now! It has been fixed will push change soon once I have the page integrated

File: web/modules/features/par_partnership_flows/src/Form/ParPartnershipFlowsInviteForm.php (hunk 3 above)
Comment (stephenb01) [reply web/modules/features/par_partnership_flows/src/Form/ParPartnershipFlowsInviteForm.php]: Agree......

Comment (stephenb01) [reply web/modules/features/par_partnership_flows/src/Form/ParPartnershipFlowsInviteForm.php]: Will the copy change for each flow instance? If so we will, at some point, need a way to change it.

**Pull request merged**